cmd/opm/registry/build.go

```diff
@@ -0,0 +1,48 @@
+package registry
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/appregistry"
+)
+
+func BuildCmd() *cobra.Command {
```

Comment:
These are not actually registered with the parent, they are inert (not just "hidden")